tests/src/rules/no-dynamic-require.js

```diff
@@ -1,10 +1,13 @@
-import { parsers, test } from '../utils';
+import { parsers, test, testVersion } from '../utils';
 
 import { RuleTester } from 'eslint';
+import flatMap from 'array.prototype.flatmap';
 
 const ruleTester = new RuleTester();
 const rule = require('rules/no-dynamic-require');
 
+const espree = require.resolve('espree');
```

Comment:
let's add this to `parsers`, or maybe better, use `null` to indicate the default?

Reply:
Thanks for review! I've added `espree` to `parsers`. a7ff5ab